README.md

```diff
@@ -1,2 +1,119 @@
-# programerski-klub-fmf-.github.io
-Super kul in dodelana stran Programerskega kluba FMF.
+# Programerski klub FMF
+
+This repository contains the code and thecontent for the official website of *Programerski klub FMF*. It relies on the *Hugo* static site generator to build the `HTML` files. It is deployed by *GitHub Pages*.
+
+## Setup
+
+Before you get started, set-up *Hugo* by following the [installation guide](https://gohugo.io/installation/). 
+
+Fork the repository, and clone the fork to your device. You can do this by running the command:
+```
+git clone "link-to-your-fork" 
+```
+
+Navigate to the directory `./your-fork-name/hugo/` with the following command:
+```
+cd your-fork-name/hugo/
+```
+
+This is where you'll do all the work. 
+
+## Adding content
+
+*Hugo* builds the sites content from `markdown` files (ending `.md`) in folder `content` and it's subfolders. Each file contains metadata as embedded `yaml` code (encapsulated by `---`), that *Hugo* can axes. 
```

Comment:
> builds the sites content
->
> builds the **site's** content

> files (ending \`.md\`) in folder \`content\` and it's subfolders.
->
> files (ending \`.md\`) in **the** folder \`content\` and **its** subfolders.

Reply:
Tu gre predvsem za to, da sem besedilo spisal ob pol eni zjutraj na telefonu kar direktno v GitHub spletni strani. Bistveno je bilo čim hitreje spraviti besedilo iz glave na stran, kar pomeni ne sproti popravljati. Torej, tu gre predvsem za napake zaradi hitrosti in ne nerazumevanja :smile: